pilot/pkg/proxy/envoy/v2/eds_test.go

```diff
@@ -620,7 +620,19 @@ func multipleRequest(server *bootstrap.Server, inc bool, nclients,
 			log.Println("Waiting for pushes ", id)
 
 			// Pushes may be merged so we may not get nPushes pushes
-			_, err = adscConn.Wait(15*time.Second, "eds")
+			got, err := adscConn.Wait(15*time.Second, "eds")
+
+			// If in incremental mode, shouldn't receive cds here
+			if inc {
+				for _, g := range got {
+					if g == "cds" {
```

Comment:
more strict check for `lds` `rds` as well

Reply:
done